my-react-app/src/app/screens/Game/layout.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Board from './components/Board';
+import styles from './styles.scss';
+
+class Game extends Component {
```

Comment:
Dumb components are usually functions.

If you don't have state and don't have lifecycle methods, you should use functional components